tests/unit_tests/test_wave_06.py

```diff
@@ -75,8 +76,14 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
+    # Assert
+    assert result is True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory == [item_d, item_e, item_c]
```

Comment:
We can't know for sure how the functions will swap the items and that this is the specific order. Comparing two lists using `==` will fail if the two lists have the same elements but they aren't in the same order. It would be better to check that each individual item is in the list. The asserts would look like this:
```
assert item_a in tai.inventory
assert item_b in tai.inventory
assert item_c not in tai.inventory
assert item_f in tai.inventory
```
and then doing the same type of asserts for Jesse.